src/main/java/frc/robot/subsystems/claw/manual/ManualConeState.java

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems.claw.manual;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+
+import edu.wpi.first.wpilibj.Timer;
+import frc.robot.Constants;
+import frc.robot.RobotMap;
+import frc.robot.lib.statemachine.State;
+import frc.robot.lib.statemachine.Transition;
+import frc.robot.subsystems.claw.ClawStateMachine;
+
+/** Add your docs here. */
+public class ManualConeState extends State {
```

Comment:
Same issues here as with the manual states in the arm, lots of code duplication.